### PR TITLE
fix(gen): force config update when it exists and `skipConfig == false`

### PR DESCRIPTION
**Pull request by kingcody**
Requires DaftMonk/generator-ng-component#21

### CODE REVIEW

File: app/generator.js [hunk 1]
```diff
@@ -49,11 +49,16 @@ export default class Generator extends Base {
...
-            this.filters = this._.defaults(this.config.get('filters'), {
-              bootstrap: true,
-              uibootstrap: true,
-              jasmine: true
-            });
+            if (this.skipConfig) {
+              this.filters = this._.defaults(this.config.get('filters'), {
+                bootstrap: true,
+                uibootstrap: true,
+                jasmine: true
```
Comment (kingcody) [inline app/generator.js]: @Awk34 I'm wondering if we even need to be setting the defaults here. I assume they are for migration purposes, can you think of any other reasons, use cases?

Comment (Awk34) [inline app/generator.js]: Hmm, I think you're right. I can't think of any other purpose either.

Comment (kingcody) [inline app/generator.js]: Here a5edbcd

Comment (Awk34) [inline app/generator.js]: Hmm, test fix? Try removing that code and running the tests.

**Comment by kingcody** [comment]
All checks out, probably since I update the fixtures config earlier to be `3.x.x` compliant.

**Comment by Awk34** [comment]
:+1:

**Pull request merged**